src/map/packets/timer_bar_util.cpp

```diff
@@ -0,0 +1,44 @@
+﻿#include "../../common/socket.h"
+
+#include "timer_bar_util.h"
+
+#include "../entities/charentity.h"
+
+CTimerBarUtilPacket::CTimerBarUtilPacket(CCharEntity* PChar)
+{
+    this->type = 0x75;
+    this->size = 0x56;
+}
+
+void CTimerBarUtilPacket::addCountdown(uint32 seconds)
+{
+    // Non-zero shows timer, zero hides it
+    ref<uint16>(0x04) = 0xFFFF;
+
+    ref<uint32>(0x08) = CVanaTime::getInstance()->getVanaTime();
+    ref<uint32>(0x0C) = seconds;
+
+    // Static data ("_unknown1" in Windower fields)
+    ref<uint8>(0x10) = 0x3C;
+    ref<uint8>(0x24) = 0x03;
+}
+
```

Comment:
Can you add another function here to set the [battlefield and render radius](https://github.com/Windower/Lua/blob/308ec1d44e94250477db634a8a0552d7e687a88a/addons/libs/packets/fields.lua#L3079-L3080) at `0x1C` and `0x20` just so it's documented and done?